src/dbnode/persist/fs/seek_manager_test.go

```diff
@@ -73,6 +74,97 @@ func TestSeekerManagerCacheShardIndices(t *testing.T) {
 	}
 }
 
+func TestSeekerManagerUpdateOpenLease(t *testing.T) {
+	defer leaktest.CheckTimeout(t, 1*time.Minute)()
+
+	var (
+		ctrl   = gomock.NewController(t)
```

Comment:
Everywhere we use mocks its best to “defer ctrl.Finish()” so that any calls you expected to happen but didn’t cause a test to fail (since this means something you expected to happen didn’t, which is a good catch).

Reply:
yeah good call, this usually happens when I copy and paste the controller code from another function I wrote in which I forgot to call finish heh